examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -586,6 +579,10 @@ extern "C" void app_main()
         return;
     }
 
+#if CONFIG_ENABLE_CHIP_SHELL
+    idf::matter::LaunchShell();
+#endif // CONFIG_ENABLE_CHIP_SHELL
+
```

Comment:
Previously the task was created after deviceManager init and InitServer calls. I assume that moving it before these won't cause any problem.

Reply:
Moving it won't cause problems. It's moved so that the heap trace extension will be initialized early enough to collect heap usage info during initialization.